shell.nix

```diff
@@ -5,6 +5,7 @@ let
 in
 
 pkgs.mkShell {
+  LOCALE_ARCHIVE = "${pkgs.glibcLocales}/lib/locale/locale-archive";
```

Comment:
This looks magical; can you add a comment explaining why this is needed (e.g. a link to https://nixos.wiki/wiki/Locales)?

Reply:
Sure